src/Microsoft.NetCore.Analyzers/Core/Security/DoNotInstallRootCert.cs

```diff
@@ -119,6 +119,17 @@ public override void Initialize(AnalysisContext context)
                     compilationStartAnalysisContext.RegisterOperationBlockStartAction(
                         (OperationBlockStartAnalysisContext operationBlockStartAnalysisContext) =>
                         {
+                            var owningSymbol = operationBlockStartAnalysisContext.OwningSymbol;
+
+                            // TODO: Handle case when exactly one of the below rules is configured to skip analysis.
```

Comment:
@dotpaul @LLLXXXCCC - I have added some TODOs for security dataflow rules that do batched DFA execution at compilation end. The DFA results only contain the location of the original call site, not the containing symbol, so it wasn't trivial for me to address the TODO.

Reply:
I will file a tracking issue for this.

Reply:
Do you mean that if the user only configured one rule to skip analysis for a given symbol, then the other rule still runs DFA, so we potentially end up with diagnostics for both rules?

Reply:
Yes, that is correct. I haven't handled that case. User would need to skip both diagnostics simultaneously.

Reply:
Filed https://github.com/dotnet/roslyn-analyzers/issues/2706